react/features/whiteboard/middleware.ts

```diff
@@ -76,6 +81,9 @@ MiddlewareRegistry.register((store: IStore) => (next: Function) => async (action
         }
 
         if (action.isOpen) {
+            if (shouldEnforceUserLimit(state)) {
```

Comment:
Shouldn't we do this above too, where we setup the whiteboard?

Reply:
If you don't have collab details, you're the first one to open it. The rest receive the collab details and go through this if. Or were you thinking about something else?

Reply:
Right, makes sense!

Reply:
Maybe I should take into account participant_left events and bump the other participants to the whiteboard if there's any room left. But this splits the participant into those who have access and those who don't, so would it be ok to keep an array of jids in the whiteboard store?

Reply:
*Only if the userLimit is present, otherwise it wouldn't make sense to keep all the jids like that

Reply:
Nah, they can press the button if they want to, IMHO. Let's not overcomplicate it for now.